doc/changes.rst

```diff
@@ -1,12 +1,302 @@
+
+``config-manager``
+  * New behavior introduced.
+  * Parameters are replaced by subcommands.
+
+    * Examples:
+
+      * Before: ``--add-repo`` Now: ``addrepo``
+      * Before: ``--save --setopt`` Now: ``setopt``
+
+  * Existing repository files are not modified; drop-in override files are created instead.
+  * See the :ref:`config-manager documentation <config_manager_plugin_ref-label>` for more information.
+
+``distro-sync``
+  * Now when any argument does not match any package or is not installed, DNF5 fail. The behavior can be modified by the ``--skip-unavailable`` option.
+  * Dropped ``distrosync`` and ``distribution-synchronization`` aliases.
+
+``downgrade``
+  * Now when any argument does not match any package or is not installed, DNF5 fail. The behavior can be modified by the ``--skip-unavailable`` option.
+
+``download``
+  * Option ``--source`` was renamed to ``--srpm``.
+  * Dropped ``--downloaddir`` argument, only ``--destdir`` is supported now.
+
+``group``
+  * New option ``--contains-pkgs`` to filter only groups containing specific packages.
+  * Dropped ``--ids`` argument as group ids are always in the output now.
+  * Dropped ``group mark install`` and ``group mark remove`` subcommands in favor of the new ``--no-packages`` option for the ``install`` and ``remove`` commands.
+
+    * E.g. to mark a group as installed without touching any packages, ``dnf5 group install --no-packages <group_id>`` command can be used.
+
+  * Dropped ``groupinstall`` alias. It is replaced by ``dnf group install``.
+  * Dropped ``groupinfo`` alias. It is replaced by ``dnf group info``.
+  * Dropped ``grouplist`` alias. It is replaced by ``dnf group list``.
+  * Dropped ``grouperase`` alias. It is replaced by ``dnf group remove``.
+  * Dropped ``groupremove`` alias. It is replaced by ``dnf group remove``.
+  * Dropped ``groupupdate`` alias. It is replaced by ``dnf group upgrade``.
+  * Dropped ``groups`` alias. It is replaced by ``dnf group``.
+
+``help``
+  * Dropped. The functionality is replaced by the ``--help`` option.
+
+``info``
+  * Dropped ``--all`` option since this behavior is the default one.
+  * Dropped ``--updates`` option, only ``--upgrades`` is available now.
+
+``list``
+  * Dropped ``--all`` option since this behavior is the default one.
+  * Changed the behavior of the ``--available`` option.
+    * In DNF4, only packages not installed or with higher versions were listed. This behavior remains unchanged when the option is not used, reducing duplications in the "Installed Packages" section.
+    * When using the ``--available`` option, DNF5 considers all versions available in enabled repositories, irrespective of the installed version.
+
+``makecache``
+  * Metadata is now stored in different directories, see the ``cachedir`` configuration option :ref:`changes <cachedir_option_conf_changes_ref-label>` for more details.
```

Comment:
The `makecache` command still exists I think this is just explanation that `dnf5` stores cache at different location.

Reply:
Yep, in this section there is always a list of changes related to an individual command. If the command is entirely dropped, it's explicitly stated here. I hope it will be better visible in the final HTML format of the docs, though there was already some confusion mentioned on the proposal discussion related to this "changes doc", so if you have a specific tip how to improve the situation, please share it with us :slightly_smiling_face: